### ISSUE TITLE
Subtitles getting out of sync when fastforwarding or resuming reproduction from an intermidiate point

### ISSUE THREAD

**Issue by gahedros** [iBaa/PlexConnect#242]
The new subtitle renderer works great in general, but I have noticed that when fastforwarding or resuming reproduction from an intermidiate point the subtitles tend to get out of sync. 
Using to the old renderer fixes the sync issues, so is not a problem related to any particular srt file.

**Comment by marcotini** [comment]
It's still open? Because even 2 years later are still a problem.

**Comment by Esp10n** [comment]
Yes I'm having the same issue here... Subtitles go out of sync....

**Comment by mderksen** [comment]
@marcotini @Esp10n : 
Open the Trailers app on aTV and go to the settings. 
Set "Transcoder Action" to AUTO
Set "Subtitle Renderer" to AUTO

This worked for me a couple of years ago :)

**Comment by chrisl31** [comment]
Helo.
I have the same issue, even after @marcotini and @Esp10n settings ...

**Comment by Esp10n** [comment]
@mderksen 
Still the same issue with those settings it's really weird....

**Comment by Ingvix** [comment]
I'm also having this issue. They also tend to go out of sync if I change them in the middle of the show. Very annoying issue.
